server/app.js

```diff
@@ -6,16 +6,14 @@ const logger = require("morgan");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
-<<<<<<< HEAD
```

Comment:
overall this is a good PR, i think we should merge this into dev because i see you resolved a merge conflict here 

i will have more feedback when this starts to come together :)